airbyte-integrations/connectors/source-recurly/source_recurly/client.py

```diff
@@ -0,0 +1,76 @@
+"""
+MIT License
+
+Copyright (c) 2020 Airbyte
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+
+import json
+import pkgutil
+from base64 import b64encode
+
+import requests
+from airbyte_protocol import AirbyteStream
+from recurly import USER_AGENT
+from recurly import Client as RecurlyClient
+from recurly.base_errors import ApiError
+
+
+class Client:
+    RECURLY_BASE_URL = "https://v3.recurly.com"
+    PAGINATION = 100
+
+    def __init__(self, api_key: str):
+        self.ENTITY_MAP = ["accounts", "coupons", "invoices", "plans", "measured_units", "subscriptions", "transactions", "export_dates"]
+        self._client = RecurlyClient(api_key)
+        self._headers = {
+            "User-Agent": USER_AGENT,
+            "Authorization": "Basic %s" % b64encode(bytes(api_key + ":", "ascii")).decode("ascii"),
+            "Accept": f"application/vnd.recurly.{self._client.api_version()}",
+            "Content-Type": "application/json",
+        }
+
+    def health_check(self):
```

Comment:
Could you use return signatures on all methods e.g: 
```suggestion
    def health_check(self) -> Tuple[Boolean, object]:
```